packages/fxa-admin-panel/src/components/AccountSearch/Subscription/index.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import dateFormat from 'dateformat';
+import { MozSubscription } from 'fxa-admin-server/src/graphql';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { ReactComponent as IconExternalLink } from '../../../images/icon-external-link.svg';
+import { DATE_FORMAT } from '../Account';
+
+const Subscription = ({
```

Comment:
Can we get a Subscription/index.test.tsx file created? It can be made robust later, but for now just to keep the test coverage stats up, it'd be nice to have a really basic 'it renders' test.